README.md

```diff
@@ -0,0 +1,1077 @@
+# Introdução
+
```

Comment:
Seria bacana usar o logo do pagar.me aqui no começo do arquivo :wink:

Reply:
Hmm, pensei que esse era o README do projeto. Se não vai ficar no repo, tudo bem, mas se o README for ficar assim, acho que seria bem massa ter o logo :)

Reply:
Sim, eu diria que é algo temporário. Faz sentido?

Reply:
Acredito que as instruções de utilização da SDK devem estar tanto na documentação do Pagar.me, quanto no README que foi criado.

Em minha visão ficaria legal o logo do Pagar.me, mas não é um impeditivo